clonelists/Nintendo - Nintendo 3DS (Digital) (CDN) (No-Intro).json

```diff
@@ -12,6 +12,20 @@
 				{"searchTerm": "Gekiyaba Runner"}

```

Comment:
Found the group `SWANG` multiple times on the following lines:

* 23
* 193

There should only be one instance of a `group` name in a `variants` array.